docs/modules/ROOT/pages/how-to/zone-setup.adoc

```diff
@@ -117,3 +117,62 @@ parameters:
 
 . Compile and push the cluster catalog
 . Wait for Argo CD to sync the config
+
+== Configure metering metrics collection
+
+We collect a small set of metrics which are used for metering from each zone.
+This is done by deploying a https://thanos.io/tip/components/receive.md/[Thanos Receiver] on each zone and configure the zone's cluster monitoring to send selected metrics to the Thanos Receiver via Prometheus's remote write feature.
+
+. Create an S3 bucket which can be accessed using the same credentials as the cluster's registry bucket.
+Use name `${CLUSTER_ID}-appuio-metering` for the bucket.
```

Comment:
small name suggestion. `metering` is for me a device or component to measure something. However the metrics we put into the bucket are already measured :)
```suggestion
Use name `${CLUSTER_ID}-appuio-metrics` for the bucket.
```

Reply:
I've used metering everywhere, since that's the term which is used in the stories.

Reply:
Well, we use both terms :shrug: 
See e.g. https://kb.vshn.ch/appuio-cloud/references/architecture/metering.html

But I don't think we should be using `metrics` and `metering` interchangeably (as synonyms). For me, metering is the whole process, whereas metrics are just the data part of metering.

@corvus-ch Any opinions?

Reply:
For me the term `metering` refers to metrics that are used for statistics and invoicing. `metrics` is more bound to metrics used for monitoring and alerting. That is just my take on it. Whatever naming we choose, we must be able to distinguish between those cases. @mhutter is working on a centralized monitoring that builds on the same tools. Those must remain separated setups and their naming should make it clear.

Reply:
Hm, if metrics is only for monitoring and alerting, wouldn't that be inconsistent with https://kb.vshn.ch/appuio-cloud/references/architecture/metrics-of-interest.html and ff?
Maybe we need to better define those terms in a separate PR.
Let's go with the `appuio-metering` bucket name for now, it's not that much of an issue for me.

Reply:
I agree. And yes, this is bottering me too.

Reply:
I think we're confusing two levels of abstraction here. On one level of abstraction we have metrics which are relevant for invoicing (what I've been calling "metering metrics" or similar in commit messages and config comments), on the other more abstract level we have the "APPUiO Metering" component (for lack of a better name).

To make it clear to which component this bucket belongs, I've been using the `appuio-metering` suffix to indicate the system component which uses this bucket.

In any case, I agree that we should probably clearly define terms around metrics and metering, but that's clearly out of scope for this PR.